app/code/Magento/Bundle/Model/Option/SaveAction.php

```diff
@@ -135,13 +147,14 @@ private function updateOptionSelection(ProductInterface $product, OptionInterfac
         if (is_array($option->getProductLinks())) {
             $productLinks = $option->getProductLinks();
             foreach ($productLinks as $productLink) {
+                $productLink->setWebsiteId($this->storeManager->getStore($product->getStoreId())->getWebsiteId());
```

Comment:
Shouldn't we add such set on the `save` method on this class as well? Or is it just relevant when we update an option?

Reply:
@gabrieldagama I believe it is only relevant when updating an option, unless I am missing something. The only time this isn't called from `save` I believe is when creating the product options, and I think you need to create them at the root level before editing any store/website scopes, correct?